### ISSUE TITLE
need SE Asia shape file

### ISSUE THREAD

**Issue by dcherian** [e-marshall/itslive#12]
oops...

In `ind_glacier_*.ipynb`: 
```
gpd.read_file('/Users/emarshall/Desktop/siparcs/data/nsidc0770_15.rgi60.SouthAsiaEast/15_rgi60_SouthAsiaEast.shp')
```

Can we replace this with an online URL?

**Comment by scottyhq** [comment]
I'd recommend converting to GeoJSON or GeoPackage https://github.com/scottyhq/rgi

Nice thing about GeoJSON is that github renders a map for you (but complains for files above 50MB, which this is)
https://github.com/scottyhq/rgi/blob/main/15_rgi60_SouthAsiaEast.geojson 

Geopackage is much more compact and is a great single-file alternative to .shps (https://www.geopackage.org). Also either geojson or geopackage can be loaded directly from URLs with geopandas so the github `raw` link: 

```python
gf = gpd.read_file('https://github.com/scottyhq/rgi/raw/main/15_rgi60_SouthAsiaEast.gpkg')
gf.head()
#            RGIId         GLIMSId   BgnDate   EndDate      CenLon  ...  TermType Surging Linkages  Name                                           geometry
#0  RGI60-15.00001  G102044E29941N  19990920  -9999999  102.044042  ...         0       9        9  None  POLYGON ((102.03759 29.93828, 102.03759 29.938...
```

**Comment by e-marshall** [comment]
added geojson to repo with db77a9ea8ce591630f78ca0d997b694c7e2e8909
updated urls in notebooks b21208a2937438ab2d5b3290456aea00fbbe6d1d

**Comment by e-marshall** [comment]
not actually working... working on a fix now

**Comment by e-marshall** [comment]
fixed with ae7598926c10a13b22349ad3d107958160a7208d
